LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
@nplasterer removed the reference to the licensing and maintenance in the README and added this standard LICENSE file. This is the one we use in xmtp-ios.